packages/modules/promotion/src/migrations/Migration20241203074045.ts

```diff
@@ -0,0 +1,167 @@
+import { Migration } from "@mikro-orm/migrations"
```

Comment:
q: anything here we should mention in the release notes?

Reply:
Nope. There are no breaking changes. Most of them are now related to FK indexes that are created automatically because of how DML setup `belongsTo` relationship